src/main/java/school/lemon/changerequest/java/introduction/hw2/Task2.java

```diff
@@ -3,6 +3,14 @@
 
 public class Task2 {
     public static String firstTwo(String s) {
-        return "";
+        if (s == null)
+            return null;
+        else if (s.equals(""))
+            return "";
+        else
+            if (s.length() <= 2) {
+                return s;
+            }
```

Comment:
If `s` will be equal to `""` than it's length will be less than 2 - so there is no need for such check.